stable/datadog/templates/agent-rbac.yaml

```diff
@@ -18,6 +18,7 @@ rules:
   - pods
   - nodes
   - componentstatuses
+  - podsecuritypolicies
```

Comment:
`podsecuritypolicies` belongs to the `policy` group

Reply:
Ah good point, thank you!